### PR TITLE
PWX-40910 | Missing default value for skipHealthChecks annotation in helm values.yaml

### PR DESCRIPTION
**Pull request by hitesh-wani-px**
**What this PR does / why we need it**:
This PR changes will add default value for skipHealthChecks flag 

**Which issue(s) this PR fixes** (optional)
Closes #https://purestorage.atlassian.net/browse/PWX-40910

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by hitesh-wani-px** [comment]
Testing Notes:
**1. Installation with 5.1.0** 
```shell
hitesh@hitesh-wani:~/go/src/github.com/portworx/helm$ helm install --debug -n kube-system --set clusterName="tp-nextpx-aks-op-helm-px-int" --set secretType="k8s" --set autopilot.enabled=true --set stork.enabled=true --set monitoring.prometheus.enabled=true --set monitoring.telemetry=true --set internalKVDB=true --set csi.enabled=true --set pxOperat
orImageVersion=24.2.1-dev --set imageVersion="9824b7b_90bc1d6" --set envVars="PX_RELEASE_MANIFEST_URL=https://edge-install.portworx.com/3.2.1.1/version"  portworx charts/portw
orx

NOTES:
Your Release is named "portworx"
Portworx Pods should be running on each node in your cluster.

Portworx would create a unified pool of the disks attached to your Kubernetes nodes. 
No further action should be required and you are ready to consume Portworx Volumes as part of your application data requirements. 

For further information on usage of the Portworx in creating Volumes please refer
    https://docs.portworx.com/portworx-enterprise/operations/operate-kubernetes/storage-operations/create-pvcs

Explore the fundamental concepts of Portworx volumes refer
    https://docs.portworx.com/portworx-enterprise/concepts

Want to use Storage Orchestration for hyperconvergence, Please look at Stork here. 
    https://docs.portworx.com/portworx-enterprise/operations/operate-kubernetes/storage-operations/stork

Track the health and performance of your storage cluster using Portworx’s monitoring features.
    https://docs.portworx.com/portworx-enterprise/operations/operate-kubernetes/monitoring

Refer to the install various stateful applications with Portworx. 
    https://docs.portworx.com/portworx-enterprise/operations/operate-kubernetes/application-install-with-kubernetes

    
For options that you could provide while installing Portworx on your cluster head over to the README.md
```
```shell
hitesh@hitesh-wani:~$ helm ls -n kube-system
NAME                            	NAMESPACE  	REVISION	UPDATED                                	STATUS  	CHART                                                                    	APP VERSION
portworx                        	kube-system	1       	2024-12-18 18:05:28.952580659 +0530 IST	deployed	portworx-5.1.0                                                           	3.2.1      
```
```shell
hitesh@hitesh-wani:~/Downloads$ kubectl get stc -n kube-system
NAME                           CLUSTER UUID                           STATUS    VERSION           AGE
tp-nextpx-aks-op-helm-px-int   bcb28d00-de8d-42cc-8631-d1afb1ea7342   Running   9824b7b_90bc1d6   7m29s

```
**1. Installation with 4.1.0** 
```shell
helm install --debug -n kube-system --set clusterName="tp-nextpx-aks-op-helm-px-int" --set secretType="k8s" --set autopilot.enabled=true --set stork.enabled=true --set monitoring.prometheus.enabled=true --set monitoring.telemetry=true --set internalKVDB=true --set csi.enabled=true --set pxOperatorImageVersion=24.2.1-dev  portworx px-4.1/portworx --version 4.1.0

NOTES:
Your Release is named "portworx"
Portworx Pods should be running on each node in your cluster.

Portworx would create a unified pool of the disks attached to your Kubernetes nodes. 
No further action should be required and you are ready to consume Portworx Volumes as part of your application data requirements. 

For further information on usage of the Portworx in creating Volumes please refer
    https://docs.portworx.com/portworx-enterprise/operations/operate-kubernetes/storage-operations/create-pvcs

Explore the fundamental concepts of Portworx volumes refer
    https://docs.portworx.com/portworx-enterprise/concepts

Want to use Storage Orchestration for hyperconvergence, Please look at Stork here. 
    https://docs.portworx.com/portworx-enterprise/operations/operate-kubernetes/storage-operations/stork

Track the health and performance of your storage cluster using Portworx’s monitoring features.
    https://docs.portworx.com/portworx-enterprise/operations/operate-kubernetes/monitoring

Refer to the install various stateful applications with Portworx. 
    https://docs.portworx.com/portworx-enterprise/operations/operate-kubernetes/application-install-with-kubernetes

    
For options that you could provide while installing Portworx on your cluster head over to the README.md
```
```shell
hitesh@hitesh-wani:~/Downloads$ helm ls -n kube-system
WARNING: Kubernetes configuration file is group-readable. This is insecure. Location: /home/hitesh/kubeconfig
WARNING: Kubernetes configuration file is world-readable. This is insecure. Location: /home/hitesh/kubeconfig
NAME    	NAMESPACE  	REVISION	UPDATED                                	STATUS  	CHART         	APP VERSION
portworx	kube-system	1       	2024-12-18 19:07:47.610829712 +0530 IST	deployed	portworx-4.1.0	3.1.7  
```
```shell
hitesh@hitesh-wani:~/go/src/github.com/portworx/helm$ kubectl get stc -n kube-system
NAME                           CLUSTER UUID   STATUS         VERSION   AGE
tp-nextpx-aks-op-helm-px-int                  Running   3.1.7     112s
```

File: charts/portworx/README.md [hunk 1]
```diff
@@ -153,7 +153,7 @@ The following tables lists the configurable parameters of the Portworx chart and
 | `updateStrategy.type` | Specifies the update strategy for the Portworx cluster. Supported values: RollingUpdate, OnDelete | "" |
 | `updateStrategy.maxUnavailable` | Maximum number of nodes that can be unavailable during a rolling update | 1 |
 | `updateStrategy.minReadySeconds` | Minimum number of seconds that a pod should be ready before the next batch of pods is updated during a rolling update | 1 |
-| `updateStrategy.disruption.allow` | This field is used to enable or disable smart and parallel upgrade. Smart upgrade is disabled by default, Enable it by setting to false. we can use the `maxUnavailable` field to control the maximum number of Portworx nodes that can be upgraded at a time | None |
+| `updateStrategy.disruption.allow` | This field is used to enable or disable smart and parallel upgrade. Smart upgrade is disabled by default, Enable it by setting to false. we can use the `maxUnavailable` field to control the maximum number of Portworx nodes that can be upgraded at a time | true |
```
Comment (nikita-bhatia) [inline charts/portworx/README.md:156]: why is this required? since by default, this can be empty as well

**Review by nikita-bhatia — APPROVED** [review]
LGTM
